core/templates/dev/head/css/oppia.css

```diff
@@ -839,6 +867,8 @@ textarea {
 
 .navbar-default .navbar-container {
   background: #009688;
+
```

Comment:
Yes I just noticed them now. I must have randomly touched them while navigating through the file some time ago. I'll have them removed with the next push.